modules/ROOT/pages/websocket/websocket-connector-cloudhub.adoc

```diff
@@ -0,0 +1,42 @@
+= Using Websockets Connector in CloudHub
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can deploy Mule apps that contain inbound WebSocket endpoints, outbound WebSocket endpoints, or both to CloudHub.
+
+If your app has inbound WebSocket endpoints only, to deploy the app in CloudHub you must:
+
+. Configure an Anypoint Virtual Private Cloud (VPC).
+. Set up a dedicated load balancer to route WS and WSS traffic through the Mule app.
+
+To set up the inbound WebSocket endpoints:
+
+. Configure the WebSockets Inbound Listener (`<ws:inbound-listener>`) to listen on one of the following ports:
+* `0.0.0.0`.
+///
```

Comment:
We should delete this /// I think it was meant to be commented out but it shows up in my local build